lib/new_relic/agent/instrumentation/active_support_subscriber.rb

```diff
@@ -2,12 +2,25 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
-require 'new_relic/agent/instrumentation/notifications_subscriber'
+require_relative 'notifications_subscriber'
 
 module NewRelic
   module Agent
     module Instrumentation
       class ActiveSupportSubscriber < NotificationsSubscriber
+        EVENT_NAME_TO_METHOD_NAME = {
+          'cache_fetch_hit.active_support' => 'fetch_hit',
+          'cache_generate.active_support' => 'generate',
+          'cache_read.active_support' => 'read',
+          'cache_write.active_support' => 'write',
+          'cache_delete.active_support' => 'delete',
+          'cache_exist?.active_support' => 'exist?',
+          'cache_read_multi.active_support' => 'read_multi',
+          'cache_write_multi.active_support' => 'write_multi',
+          'cache_delete_multi.active_support' => 'delete_multi',
+          'message_serializer_fallback.active_support' => 'message_serializer_fallback'
```

Comment:
When I was looking at the docs, in the '[3.8 Active Support — Messages](https://edgeguides.rubyonrails.org/active_support_instrumentation.html#active-support-%E2%80%94-messages)' section, (anchor link might not work) it seems like the payload is pretty different from the payload for the caching events. I wonder if we should create a different instrumentation module for message notifications, even though there's only one.

Instead of key/store/etc. The `message` notification's payload includes serializer/fallback/serialized/deserialized

Reply:
Created #2615

Reply:
Thank you!